samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/SubmitToGradingActionListener.java

```diff
@@ -657,7 +657,7 @@ private void prepareItemGradingPerItem(ActionEvent ae, DeliveryBean delivery,
 							|| itemgrading.getAttemptDate() != null) { 
 							// null=> skipping this question
 							itemgrading.setAgentId(AgentFacade.getAgentString());
-							itemgrading.setSubmittedDate(new Date());
+							verifySubmittedDateInItemGrading(item, itemgrading);
```

Comment:
```
itemgrading = verifySubmittedDateInItemGrading(item, itemgrading);
```

verifySubmittedDateInItemGrading method return an ItemGradingData

Reply:
Yeah, it is unnecesary, because it is modified in the method, there is no need of setting it, i Will change the method as void